main.go

```diff
@@ -86,6 +92,7 @@ func main() {
 	flag.StringVar(&tokenFile, "token-file", defaultTokenFile, "The path of the service account token.")
 	flag.StringVar(&caCertFile, "ca-cert-file", defaultCACertFile, "The path of the CA cert of the Agents' signing key pair.")
 	flag.BoolVar(&enableLeaderElection, "leader-elect", false, "Enable leader election for controller manager. "+"Enabling this will ensure there is only one active controller manager.")
+	flag.BoolVar(&enableWebhook, "enable-webhook", false, "Enable the webhook server(s). Defaults to true.")
```

Comment:
```suggestion
	flag.BoolVar(&enableWebhook, "enable-webhook", true, "Enable the webhook server(s). Defaults to true.")
```

Reply:
This was done on purpose, I didn't want to start webhook server until the conversion config is not there yet. Will do it [in the followup PR](https://github.com/openshift/node-observability-operator/pull/91).

Reply:
ahh, that makes sense.